task/buildah-oci-ta/0.2/buildah-oci-ta.yaml

```diff
@@ -242,6 +242,8 @@ spec:
         - name: COMMIT_SHA
           value: $(params.COMMIT_SHA)
       script: |
+        #!/bin/bash
+        set -e
```

Comment:
Why the addition of `-e`?

Reply:
Ah, this is re-added by the trusted artifact generation. :)

Reply:
We have seen scripts encounter errors but not fail when 'set -e' is not there.